ndbench-dynamodb-plugins/build.gradle

```diff
@@ -4,10 +4,9 @@ repositories {
    }
 }
 
-
 dependencies {
     compile project(':ndbench-api')
-    compile "com.amazonaws:aws-java-sdk-dynamodb:latest.release"
+    compile "software.amazon.awssdk:dynamodb:latest.release"
```

Comment:
Might you please educate me on the difference?

Reply:
Using AWS SDK 2.0 will allow the use of a Netty async HTTP client. It is my understanding that we might be able to get more TPS with fewer HTTP connections using Netty. AWS SDK 2.0 is still in public beta, so this pull request is lower priority.